src/main/java/mchorse/mappet/entities/EntityNpc.java

```diff
@@ -66,6 +67,11 @@ public class EntityNpc extends EntityCreature implements IEntityAdditionalSpawnD
     public float smoothBodyYawHead;
     public float prevSmoothBodyYawHead;
 
+    /*
```

Comment:
It should be /** because JavaDoc, plus there are more than one space in the comment.

Reply:
Fixed